### DISCUSSION TITLE
mcp23017 problem

### DISCUSSION THREAD

**Discussion by szybki946** [lkankowski/arduino-multi-relay2#48] [General]
w nowym kodzie wyskakuje błąd:
compilation terminated.
*** [.pio\build\mega2560-serial\libd59\Adafruit MCP23017 Arduino Library\Adafruit_MCP23X17.cpp.o] Error 1
In file included from .pio\libdeps\mega2560-serial\Adafruit MCP23017 Arduino Library\src\Adafruit_MCP23XXX.cpp:23:0:
.pio\libdeps\mega2560-serial\Adafruit MCP23017 Arduino Library\src\Adafruit_MCP23XXX.h:8:10: fatal error: Adafruit_BusIO_Register.h: No such file or 
directory

**Comment by lkankowski** [comment]
Jaka nowa wersja? Ostatnia zmiana była ponad rok temu :D
- Jaki branch? (domyślny: master)
- jaki target kompilacji
- jaki config
itp.

U mnie kompilacja `mega2560-serial-mcp23017` lub `esp8266-wifi-mcp23017` przebiega bezproblemowo.
